### PR TITLE
Set timestamp in upload

### PR DESCRIPTION
**Pull request by ylangisc**

### CODE REVIEW

File: eue/src/main/java/ch/cyberduck/core/eue/EueMultipartWriteFeature.java [hunk 1]
```diff
@@ -96,8 +96,8 @@ public HttpResponseOutputStream<EueWriteFeature.Chunk> write(final Path file, fi
             throw new ChecksumException(LocaleFactory.localizedString("Checksum failure", "Error"), e);
         }
         return new HttpResponseOutputStream<EueWriteFeature.Chunk>(new MemorySegementingOutputStream(proxy,
-                new HostPreferences(session.getHost()).getInteger("eue.upload.multipart.size")),
-                new EueAttributesAdapter(), status) {
+            new HostPreferences(session.getHost()).getInteger("eue.upload.multipart.size")),
```
Comment (AliveDevil) [inline eue/src/main/java/ch/cyberduck/core/eue/EueMultipartWriteFeature.java:99]: this constant reindenting is super annoying, can we get some unified code styles enforced?

**Review by AliveDevil — APPROVED** [review]

File: eue/src/main/java/ch/cyberduck/core/eue/EueMultipartWriteFeature.java (hunk 1 above)
Comment (ylangisc) [reply eue/src/main/java/ch/cyberduck/core/eue/EueMultipartWriteFeature.java:99]: Agree. We have `.idea/codeStyleSettings.xml` which should ensure a unified style. Behaviour changes from version to version. Should probably open an issue with the JetBrains guys.

Comment (AliveDevil) [reply eue/src/main/java/ch/cyberduck/core/eue/EueMultipartWriteFeature.java:99]: Maybe there are settings in codeStyleSettings that are deprecated/replaced/removed, and user-defaults are taken, which differ from those originally set in codeStyleSettings?

**Review by dkocher — APPROVED** [review]

**Review by dkocher — CHANGES_REQUESTED** [review]
Failing testcase.

```
java.lang.AssertionError: 
Expected :1650100574172
Actual   :1650100575347
<Click to see difference>


	at org.junit.Assert.fail(Assert.java:89)
	at org.junit.Assert.failNotEquals(Assert.java:835)
	at org.junit.Assert.assertEquals(Assert.java:647)
	at org.junit.Assert.assertEquals(Assert.java:633)
	at ch.cyberduck.core.eue.EueWriteFeatureTest.testWrite(EueWriteFeatureTest.java:140)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater$1.execute(IdeaTestRunner.java:38)
	at com.intellij.rt.execution.junit.TestsRepeater.repeat(TestsRepeater.java:11)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:35)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
```

**Comment by ylangisc** [comment]
As mentioned in the ticket the backend change has not been deployed yet.

**Comment by dkocher** [comment]
> As mentioned in the ticket the backend change has not been deployed yet.

Seems to be deployed now but gives error 

```
{"status":400,"type":"urn:problem:ui:malformed-request-data","exceptionClass":"MismatchedInputException"}
```

**Review by dkocher — APPROVED** [review]

**Pull request merged**